components/digital_ocean/actions/add-ssh-key/add-ssh-key.mjs

```diff
@@ -1,45 +1,31 @@
-// legacy_hash_id: a_8KiVPz
-import doWrapperModule from "do-wrapper";
+import digitalOceanApp from "../../digital_ocean.app.mjs";
 
 export default {
   key: "digital_ocean-add-ssh-key",
   name: "Add SSH Key",
   description: "Adds a new SSH to your account.",
-  version: "0.1.1",
+  version: "0.1.2",
   type: "action",
   props: {
-    digital_ocean: {
-      type: "app",
-      app: "digital_ocean",
-    },
-    page_size: {
-      type: "integer",
-      description: "Desired pagination size when pulling results",
-      optional: true,
-    },
+    digitalOceanApp: digitalOceanApp,
     name: {
+      label: "Name",
       type: "string",
-      description: "The name to give the new SSH key in your account.",
+      description: "A human-readable display name for this key, used to easily identify the SSH keys when they are displayed.",
     },
     public_key: {
+      label: "Public key",
       type: "string",
-      description: "A string containing the entire public key.",
+      description: "The entire public key string. Embedded into the root user's authorized_keys file if you include this key during Droplet creation.",
     },
   },
   async run({ $ }) {
-    var DigitalOcean = doWrapperModule.default,
-      api = new DigitalOcean(this.digital_ocean.$auth.oauth_access_token, this.page_size);
-
-    try {
-      var configuration = {
-        "name": this.name,
-        "public_key": this.public_key,
-      };
-
-      $.export("resp", await api.accountAddKey(  configuration ));
-    } catch (err) {
-      $.export("err", err);
-
-    }
+    const api = this.digitalOceanApp.digitalOceanWrapper();
+    var newKeyData = {
+      name: this.name,
+      public_key: this.public_key,
+    };
+    $.export("newKeyData", newKeyData);
+    return await api.keys.add(newKeyData);
```

Comment:
Please add a better summary for users here and in the rest of components for this app
```suggestion
    const respone = await api.keys.add(newKeyData);

    $.export("$summary", `Successfully added SSH key with name ${this.name}`);

    return respone;
```